.github/workflows/github-actions-unit-test.yml

```diff
@@ -67,16 +67,52 @@ jobs:
       - name: Unit Test NeuroId (ReactNative)
         run: ./gradlew :NeuroID:testReactNativeLibDebugUnitTest
 
-        # Execute unit tests
-      - name: Unit Test
+  android_sdk_integration_test_layout_app:
+    runs-on: [macos-12]
```

Comment:
Don't see a reason to use macos. you can change this to ubuntu-latest.

Reply:
According to this article, https://medium.com/@esthcarelle/continuous-delivery-for-android-using-github-actions-effortless-android-testing-building-and-d3759ce66895, the reactivecircus emulator uses hardware acceleration only on MacOS which seems to make the tests more reliable which is why I moved these to separate sections to run the integration tests in MacOS. With ubuntu, this emulator seems to start up more times then not causing the tests to fail out with the "cannot find root window...." error.